src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -765,6 +767,15 @@ private function registerAssetsConfiguration(array $config, ContainerBuilder $co
             $defaultVersion = $this->createVersion($container, $config['version'], $config['version_format'], '_default');
         }
 
+        if (class_exists(PreloadManager::class)) {
```

Comment:
I don't get what you mean. But if it's ok to bump the dependency, I'll move this definition to the XML, it's cleaner.